### ISSUE TITLE
Unclear documentation on finding & updating package

### ISSUE THREAD

**Issue by jfro** [MicrosoftDocs/msix-docs#185]
We're currently using MSIX & AppInstaller file for limited out of app store distribution, and wanted to add some in-app checking for updates functionality. Problem is the examples in the documentation aren't clear or helpful and a lot of experimentation seemed required to finally arrive at a semi-working point.

There's 2 big hiccups we ran into:

- `targetPackageFullName` it wasn't mentioned at all what this should be. It took multiple tries until I landed on the full version included package name: `OurCompany.OurApp_0.1.13.0_x64__hashidthing` which seems inconvenient, or at least unclear how to obtain it without some work to build it from current app's version etc.
- `ConsolidateAppInstallerView()` isn't explained, and doesn't seem to be part of the windows SDK, so not sure what the expectation is if it's something we should define.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 61dbc475-482a-3e2f-b41e-bb9d35d100e5
* Version Independent ID: 93e1b53d-8150-5693-ca85-acb317f1f666
* Content: [Update non-Store published apps from your code - MSIX](https://docs.microsoft.com/en-us/windows/msix/non-store-developer-updates)
* Content Source: [msix-src/non-store-developer-updates.md](https://github.com/microsoftdocs/msix-docs/blob/master/msix-src/non-store-developer-updates.md)
* Product: **msix**
* GitHub Login: @Huios
* Microsoft Alias: **diahar**

**Comment by Berglund81** [comment]
@jfro I agree with you in you points! About the targetPackageFullName if you run this code from the same app that you want to update I found that you can use `Package.Current.Id.FullName` to get the value for the app. But that is only if you run the update function from the same app which you want to update, so I don´t know if this helps you.

**Comment by RDMacLachlan** [comment]
Updated documentation to include a note on `targetPackageFullName`. 

Branch: User/romaclac/Bug/Unclear-documentation-value-descriptions
